omod/src/main/java/org/openmrs/module/idgen/rest/resource/IdentifierResource.java

```diff
@@ -102,7 +102,7 @@ public Identifier newDelegate() {
 
 	@Override
 	public Identifier save(Identifier delegate) {
-		throw new ResourceDoesNotSupportOperationException();
```

Comment:
Any reason why you are some times using ResourceDoesNotSupportOperationException and other times UnsupportedOperationException?

Reply:
Well your right on this. This is a tweak or hack around to make the POST path for this resource be added in the docs. Well look at this [method that determines supported operations](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java#L331). I assumes a Resource that supports `#create()` should also support `#save()` operations which isn't the case for this resource. That's why I intended to throw UnsupportedOperationException which is an instance of ResponseException but is not checked by the `#testOperationImplemented()` in the webservices.rest module. This way the POST path can be included in the docs. The other alternative was to just `return null;` but I found it trivial

Reply:
Doesn't it make more sense to instead fix testOperationImplemented()?

Reply:
Well it makes more sense and meaning but I believe that is out of scope for this ticket. I suppose it shouldn't block closure of this issue just according to our RoadMap  :)

Reply:
Why would we hack this module instead of just fixing the other?